### DISCUSSION TITLE
Couldn't figure out what is purpose of certificated.py in Server Project

### DISCUSSION THREAD

**Discussion by dungdamquoc1993** [Y4hL/PyDoor#15] [Q&A] [answered]
I found that in the Esocket.py everything is already encrypted just don't know why certificated.py exist

**Comment by tidely** [comment, marked as answer]
ESocket is currently set up to create a encrypted connection between two endpoints. That being said, there is currently no way of verifying the identity of a peer. That is what certificates are for. Although there is no support for certificates in the handshake, meaning currently the certificates.py file only exists for development purposes.

**Reply by dungdamquoc1993** [reply]
Oh, Thank You so much I highly appreciate your quick response. The code look very nice
